src/get-started/build-tidb-from-source.md

```diff
@@ -3,8 +3,17 @@
 ## Prerequisites
 
 * `git`: The source code of TiDB is hosted on GitHub as a git repository. To work with git repository, please [install `git`](https://git-scm.com/downloads).
-* `go`: TiDB is a Go project thus its building requires a working `go` environment. See the previous [Install Golang](https://github.com/zz-jason/tidb-dev-guide/tree/ac6b65f27b589a78fff1767e94703eb1ca25874e/get-started/get-started/install-golang.md) section to prepare the environment.
-* `mysql` client \(optional\): After building TiDB from source, you can use use the official [MySQL client](https://dev.mysql.com/downloads/mysql/) to connect to TiDB. It is not required if you want to build TiDB only.
+* `go`: TiDB is a Go project thus its building requires a working `go` environment. See the previous [Install Golang](install-golang.md) section to prepare the environment.
+* `mysql` client (optional): After building TiDB from source, you can use use the official [MySQL client](https://dev.mysql.com/downloads/mysql/) to connect to TiDB. It is not required if you want to build TiDB only.
+* `WSL2` (Win10 only): TiDB should compile and run on Windows 10, but it is not expected to be deployed on Windows, where you might encounter many compatibility problems. To have a better experience, it is recommended to [install WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10) first. 
```

Comment:
```suggestion
* `WSL2` (Win10 only): TiDB should compile and run on Windows 10, but it is not expected to be deployed on Windows, where you might encounter many compatibility problems. To have a better experience, it is recommended to [install WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10) first.
```
Please remove the tailing space.

Reply:
IMHO it's weird to list "WSL2" here since it is just one solution for Windows users.

Can we set a new 'tips' section for Windows users instead?

```
* `go`: ...
* `mysql` client (optional): ...

Tips: For Windows users, ..., it is recommended to use the [Windows Subsystem for Linux](...)...
```

Reply:
Make since.